commit c3b33b2df977305c54319c8648bcfc5932074f75
Author: Jonas Jelten <jj@sft.mx>
Date:   Sun Aug 20 00:30:16 2017 +0200

    fix installation path for library
    
    `lib` may be wrong on multiarch-systemd like Gentoo or Debian.
    This dynamically adjusts the target directory to be `lib64` or `lib`.

diff --git a/rct.cmake b/rct.cmake
index 46c9607..323e7b9 100644
--- a/src/rct/rct.cmake
+++ b/src/rct/rct.cmake
@@ -19,6 +19,7 @@ include(CheckSymbolExists)
 include(CheckCXXSymbolExists)
 include(CheckCXXSourceCompiles)
 include(CheckCXXSourceRuns)
+include(GNUInstallDirs)
 
 include(${CMAKE_CURRENT_LIST_DIR}/compiler.cmake)
 set(CMAKE_MODULE_PATH ${CMAKE_OLD_MODULE_PATH})
@@ -235,7 +236,7 @@ endif ()
 
 if (NOT RCT_NO_INSTALL)
   install(CODE "message(\"Installing rct...\")")
-  install(TARGETS rct DESTINATION lib COMPONENT rct EXPORT rct)
+  install(TARGETS rct EXPORT rct LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR})
 endif ()
 
 set(CMAKE_REQUIRED_FLAGS "-std=c++11")
